bundles/org.openhab.binding.denonmarantz/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -11,6 +11,25 @@
 			</update-channel>
 		</instruction-set>
 
+		<instruction-set targetVersion="2">
+			<update-channel id="power" groupIds="general">
+				<type>system:power</type>
+				<description>Power ON/OFF the AVR</description>
+			</update-channel>
+			<update-channel id="power" groupIds="mainZone,zone2,zone3,zone4">
```

Comment:
I assume you tested it is working when you provide a comma-separated list of groups ?

Reply:
Yes, it is tested and working.